bigip/resource_bigip_ltm_node.go

```diff
@@ -183,6 +180,8 @@ func resourceBigipLtmNodeRead(d *schema.ResourceData, meta interface{}) error {
 
 	d.Set("connection_limit", node.ConnectionLimit)
 	d.Set("dynamic_ratio", node.DynamicRatio)
+	d.Set(("fqdn.0.interval"), node.FQDN.Interval)
```

Comment:
This is an element in an aggregate type, I believe this should be error checked when refreshing state.